includes/modules/new_products.php

```diff
@@ -94,7 +94,7 @@
     }
 
     if (!empty($new_products_category_id)) {
-        $category_title = zen_get_category_name((int)$new_products_category_id);
+        $category_title = zen_get_category_name((int)$current_category_id);
```

Comment:
Should we be checking `current_category_id` in the if block above?

Reply:
yes have changed logic